core/src/main/scala/org/broadinstitute/dsde/rawls/entities/EntityManager.scala

```diff
@@ -49,7 +49,12 @@ class EntityManager(providerBuilders: Set[EntityProviderBuilder[_ <: EntityProvi
 
   def resolveProvider(requestArguments: EntityRequestArguments): Try[EntityProvider] = {
 
-    if (!WorkspaceType.RawlsWorkspace.equals(requestArguments.workspace.workspaceType)) {
+    // Azure workspaces use a WDS application in the data plane, Rawls entity service requests are not supported.
+    if (
+      WorkspaceType.McWorkspace.eq(
```

Comment:
Thinking out loud, wondering what might be a good way to represent the "flavor" of workspace in the Rawls model. I guess we have 3 cases, right?
1. Rawls workspace - GCP
2. MC workspace - GCP
3. MC workspace - Azure

Reply:
It's seems like there's a 4th one, the Rawls w/"Rawls Stage" MC Workspace.

Reply:
*nod *

I wonder if it's worth explicitly enumerating all the cases, instead of relying on combinations of certain fields to distinguish them.

Reply:
Yeah, it's admittedly awkward. I'll see about evolving the data model.